microprofile-jwt/src/test/java/org/jboss/eap/qe/microprofile/jwt/compatibility/package-info.java

```diff
@@ -0,0 +1,4 @@
+/**
+ * Set of tests verifying specific specification compatibility related functions.
+ */
+package org.jboss.eap.qe.microprofile.jwt.compatibility;
```

Comment:
Create an issue for this so we can have and task to document other packages.

Reply:
I think I'm missing here something. Are we required to add `package-info.java ` when providing JavaDoc for test plan generator?

Reply:
AFAIK no, we are not, this is just for future reference and my own initiative.

Reply:
ok, I guess there should be issue added ;-)